<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>720</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>串口调试助手</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>串口</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>串口号</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QComboBox" name="comboBoxSerialPort"/>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>波特率</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QComboBox" name="comboBoxBaud"/>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>数据位</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QComboBox" name="comboBoxDateNum"/>
             </item>
             <item row="3" column="0" colspan="2">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>校验位</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QComboBox" name="comboBoxCheckBit"/>
             </item>
             <item row="4" column="0" colspan="2">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>停止位</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QComboBox" name="comboBoxStopBit"/>
             </item>
             <item row="5" column="2">
              <widget class="QPushButton" name="pushButtonSerialOperation">
               <property name="text">
                <string>打开串口</string>
               </property>
               <property name="checkable">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0" colspan="2">
              <widget class="QLabel" name="labelSerialOperation">
               <property name="text">
                <string>操作</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>接受区设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButton">
             <property name="text">
              <string>清空</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QCheckBox" name="checkBoxHexReceiveShow">
             <property name="text">
              <string>十六进制显示</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QSpinBox" name="spinBoxFontSize">
             <property name="minimum">
              <number>5</number>
             </property>
             <property name="maximum">
              <number>30</number>
             </property>
             <property name="singleStep">
              <number>1</number>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>字体大小</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxTime">
             <property name="text">
              <string>时间戳</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>字体颜色</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>黑</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>蓝</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>红</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>绿</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>发送区设置</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QCheckBox" name="checkBoxHexSendShow">
             <property name="text">
              <string>十六进制显示</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="checkBoxShowSendData">
             <property name="text">
              <string>显示发送字符串</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="title">
           <string>换行</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="1">
            <widget class="QComboBox" name="comboBoxNextLine">
             <property name="currentIndex">
              <number>2</number>
             </property>
             <item>
              <property name="text">
               <string>\r\n(CRLF)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>\r(CR)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>\n(LF)</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>换行符</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>GroupBox</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_8">
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonSend">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="MinimumExpanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>发送</string>
             </property>
             <property name="icon">
              <iconset>
               <normalon>:/image/send.png</normalon>
              </iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>64</width>
               <height>64</height>
              </size>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit"/>
           </item>
          </layout>
         </widget>
        </item>
        <item row="0" column="1" rowspan="3">
         <widget class="QPlainTextEdit" name="plainTextEdit"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFileSave"/>
   <addaction name="actionAbout"/>
   <addaction name="actionAboutQt"/>
  </widget>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="text">
    <string>AboutQt</string>
   </property>
  </action>
  <action name="actionFileSave">
   <property name="text">
    <string>FileSave</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
